packages/sdk/src/endpoints/AttributesEndpoint.ts

```diff
@@ -22,4 +26,12 @@ export class AttributesEndpoint extends Endpoint {
     public async getValidAttributes(request: GetValidAttributesRequest): Promise<ConnectorResponse<ConnectorAttribute>> {
         return await this.get("/api/v1/Attributes/Valid", request);
     }
+
+    public async executeIdentityAttributeQuery(query: ExecuteIdentityAttributeQueryRequest): Promise<ConnectorResponse<ConnectorAttribute>> {
```

Comment:
Als Methodenname hast du `executeIdentityAttributeQuery`, in der Route aber `ExecuteIdentityQuery`. Ähnlich bei Relationship. Ist das Absicht? Wenn ja wieso?

Reply:
Ich glaube, dass ExecuteIdentityQuery etwas irreführend ist. Das suggeriert irgendwie, dass damit eine Identity gequeried wird.

Reply:
Stimmt, ich bau es beim nächsten PR mit ein